pkg/controller/constrainttemplate/constrainttemplate_controller_test.go

```diff
@@ -522,9 +522,9 @@ func TestReconcile(t *testing.T) {
 		}
 	})
 
-	t.Run("VapBinding should be created with v1beta1", func(t *testing.T) {
-		suffix := "VapBindingShouldBeCreatedV1Beta1"
-		logger.Info("Running test: VapBinding should be created with v1beta1")
+	t.Run("VapBinding should not be created without VAP", func(t *testing.T) {
+		suffix := "VapBindingShouldNotBeCreatedWithoutVAP"
+		logger.Info("Running test: VapBinding should be created without VAP")
```

Comment:
mismatch here. is it "should create" or "should not create"?

Reply:
It shoul be "should not create"